### PR TITLE
Testing : 6.0.2 + packaging v2 + fix install 502 error + timeout on slow hardware + wait for DB full init

### PR DESCRIPTION
**Pull request by ericgaspar**
- #400 
- #397 
- #399 
- ?

## PR Status

- [ ] Code finished and ready to be reviewed/tested
- [ ] The fix/enhancement were manually tested (if applicable)

## Automatic tests

Automatic tests can be triggered on https://ci-apps-dev.yunohost.org/ *after creating the PR*, by commenting "!testme", "!gogogadgetoci" or "By the power of systemd, I invoke The Great App CI to test this Pull Request!". (N.B. : for this to work you need to be a member of the Yunohost-Apps organization)

### CODE REVIEW

**Comment by ericgaspar** [comment]
!testme

**Comment by alexAubin** [comment]
Meh, I dont understand why it 502 yet the service seems running and no error can be found in logs x_x

**Comment by ericgaspar** [comment]
!testme

**Comment by ericgaspar** [comment]
!testme

File: scripts/install [hunk 1]
```diff
@@ -163,6 +163,7 @@ ynh_script_progression --message="Building Yarn dependencies..."
 
 pushd "$final_path"
 	ynh_use_nodejs
+	npm install -g npm
```
Comment (lapineige) [inline scripts/install:166]: Isn't that bad practice to install globally ? :thinking:

**Comment by lapineige** [comment]
!testme

File: scripts/install (hunk 1 above)
Comment (lapineige) [reply scripts/install:166]: ping @alexAubin

Comment (lapineige) [reply scripts/install:166]: #402 to test without it

**Comment by lapineige** [comment]
!testme

**Comment by alexAubin** [comment]
!testme

**Review by alexAubin — APPROVED** [review]

**Comment by alexAubin** [comment]
Sooo tests are passing, for me we're good to go :+1:

**Comment by lapineige** [comment]
That install didn't work on my server… I already had a report of a failed update.

A so massive change requires manual testing imho…

**Comment by alexAubin** [comment]
It it fails then can we have logs x_x ...

**Comment by lapineige** [comment]
So I tried again (on 2 instances) with your latest changes, and it works now. I don't know what fixed it.

But for someone else, it failed. Logs : https://paste.yunohost.org/raw/yecofedamu

**Comment by ericgaspar** [comment]
I have pushed to testing some fixes for the NGINX config:
 `__DATA_DIR__storage/` to `__DATA_DIR__/storage/`
I don't know if this solve issues beeing reported in the forum.

**Comment by lapineige** [comment]
So referencing it https://github.com/YunoHost-Apps/peertube_ynh/pull/404

**Pull request merged**